ksqldb-serde/src/main/java/io/confluent/ksql/serde/protobuf/ProtobufSchemas.java

```diff
@@ -23,12 +23,21 @@ public final class ProtobufSchemas {
   private ProtobufSchemas() {
   }
 
+  /**
+   * Adds the {@code schemaName} to the provided {@code schema}. The name is only added to the
+   * parent schema name to allow Protobuf to extract the name later. Support for nested schema
```

Comment:
What does it mean that nested schema extractions are not supported? (What is a nested schema extraction?)

Reply:
I should have added a better comment :). But multiple schemas definitions may be available in SR, like:
```
syntax = "proto3";

message Key1 {
  int32 ID = 1;
}

message Key2 {
  string ID = 1;
}

message Key3 {
  double ID = 1;
}
```

If a schema name `Key2` is specified, then the SR API will extract the following schema from the above:
```
syntax = "proto3";

message Key2 {
  string ID = 1;
}
```

The comment mentions this is only allowed for a parent schema name like the above examples. A nested schema name (not supported) would be `Key2.InternalKey`, like below:
```
syntax = "proto3";

message Key2 {
  string ID = 1;
  InternalKey IK = 2;

  message InternalKey { ... }
}
```